pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -1185,22 +1185,50 @@ public void testQueryWithOrderby()
   }
 
   @Test
-  public void testQueryWithSameAlias()
+  public void testQueryWithAlias()
```

Comment:
Do we have the test for `select *` ?

Reply:
There are some other tests in the class to cover `select *`.